File: Java/DayTwo/FireDrill/QuestionOne.java
import java.util.Scanner;
public class QuestionOne {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);
System.out.println("Enter an integer");
int firstNumber = input.nextInt();
System.out.println("Enter an integer");
int secondNumber = input.nextInt();
//System.out.println("Enter an integer");
//int thirdNumber = input.nextInt();

int first_integer_squared = firstNumber * firstNumber;
int second_integer_squared = secondNumber * secondNumber;

int sum_squared = first_integer_squared + second_integer_squared;
int difference_of_squares = first_integer_squared - second_integer_squared;

System.out.printf("The square of: %d is %d and %d is %d, the sumof their squares is %d while the difference is %d ",firstNumber,first_integer_squared,secondNumber,second_integer_squared,sum_squared,difference_of_squares);
}
}

File: JavaScript/DayTwo/FireDrill/question_one.js
const prompt = require("prompt-sync")();
let first_integer = parseInt(prompt("Enter An Integer: "))
let second_integer = parseInt(prompt("Enter An Integer: "))

first_integer_squared = first_integer **2
second_integer_squared = second_integer ** 2

let sum_squared = first_integer_squared + second_integer_squared
let difference_of_squares = first_integer_squared - second_integer_squared

console.log("The square of: " + first_integer + " is " + first_integer_squared + " and " + second_integer + " is " + second_integer_squared + " the sumof their squares is " + sum_squared + " while the difference is " + difference_of_squares)
